### PR TITLE
Added UI for checkbox in about view

### PR DESCRIPTION
**Pull request by dolftax**

### CODE REVIEW

File: css/switches.css [hunk 1]
```diff
@@ -1,58 +1,55 @@
-.cmn-toggle {
-    position: absolute;
-    margin-left: -9999px;
-    visibility: hidden;
+input.switch:empty
```
Comment (kaustavdm) [inline css/switches.css]: Conform to the style on the rest of the CSS, please. The declaration should open on the same line.

**Comment by kaustavdm** [comment]
@dolftax I know this has been waiting for a looooong time. Nothing wrong with this. But I am too busy. Let's just hold this for a few more days till I get back. :+1:

**Comment by kaustavdm** [comment]
Okay, it's finally in.

**Comment by dolftax** [comment]
Those semi-noob days!

**Pull request merged**